CMakeLists.txt

```diff
@@ -29,6 +29,11 @@ endif()
 set(PROFILE_GPERFTOOLS "${PROFILE_GPERFTOOLS}"
   CACHE BOOL "Build with the GPerfTools profiler library" FORCE)
 
+if(NOT BUILD_BENCHMARK_LOG)
+  set(BUILD_BENCHMARK_LOG OFF)
+endif()
+set(BUILD_BENCHMARK_LOG "${BUILD_BENCHMARK_LOG}"
+  CACHE BOOL "Build binary which benchmarks spdlog speed" FORCE)
```

Comment:
Very trivial point, but can you change `which` to `that` (it's a defining clause)